src/main/java/dk/kb/datahandler/preservica/client/DsPreservicaClient.java

```diff
@@ -318,4 +319,64 @@ public InputStream getFileRefForContentObject(String id) throws IOException, URI
         return connection.getInputStream();
     }
 
+    /**
+     * Call the endpoint {@code /api/entity/information-objects/{id}/identifiers}
+     * @param id of the information object to retrieve identifiers for.
+     * @return an input stream containing the response from the API.
+     */
+    public InputStream getIdentifiers(String id) throws URISyntaxException, IOException{
```

Comment:
Double client code.
Also name getIdentifiers not  obvius since it returns a InputStream abnd require work to get identifiers.
Extract to one methoed that takes pathSegment as input also. Only difference I can see.

Reply:
Ive chosen the name as the endpoint it calls is a GET request to `/identifiers`. Do you have a suggestion for a better name? getIdentifiersAsStream maybe?